integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/preaggregate/TestPreAggCreateCommand.scala

```diff
@@ -377,11 +379,15 @@ class TestPreAggCreateCommand extends QueryTest with BeforeAndAfterAll {
     sql("DROP DATAMAP IF EXISTS agg0 ON TABLE maintable")
   }
 
-  test("test show tables filterted with datamaps"){
+  test("test show tables filterted with datamaps") {
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_SHOW_DATAMAPS,"false")
     sql("create table showTables(name string, age int) stored by 'carbondata'")
-    sql("create datamap preAgg on table showTables using 'preaggregate' as select sum(age) from showTables")
+    sql(
+      "create datamap preAgg on table showTables using 'preaggregate' as select sum(age) from showTables")
     sql("show tables").show()
-    assert(!sql("show tables").collect().contains("showTables_preagg"))
+    checkExistence(sql("show tables"), false, "showtables_preagg")
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_SHOW_DATAMAPS,CarbonCommonConstants.CARBON_SHOW_DATAMAPS_DEFAULT)
```

Comment:
Keep it finally block

Reply:
reset the property to default in afterAll() block